File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 Modul 6/Python/PRAK603-2410817110006-HamkaArifani.py
a,b=map(int,input("").split())
if a != b:
    print("Jumlah tidak sama")
else:
    kolom1=list(map(int,input("").split()))
    kolom2=list(map(int,input("").split()))
    for i in range (a):
        print(kolom1[i]*kolom2[i],end=' ')

File: Praktikum Pemrograman 1/Praktikum Pemrograman 1 Modul 6/C/PRAK603-2410817110006-HamkaArifani.c
#include<stdio.h>
int main(){
    int a, b;
    scanf("%d %d", &a, &b);
    if (a!=b){
        printf("Jumlah tidak sama");
    }
    else{ 
      int kolom1[a],kolom2[b];
      for(int i=0;i<a;i++){
        scanf("%d ", &kolom1[i]);
         }
        for (int i=0;i<b;i++){
        scanf("%d", &kolom2[i]);
        }
        for (int i=0;i<a;i++){
            printf("%d ",kolom1[i]*kolom2[i]);
        }
      }
}